src/services/openfin-service.js

```diff
@@ -9,7 +9,7 @@
                 storeService.incrementStock(title);
                 var child = new fin.desktop.Window({
                     name: title + ' Stock Data',
-                    url: 'd3fc-showcase.html'
+                    url: 'd3fc-showcase.html?stockcode='+title
                 }, function() {
```

Comment:
Wouldn't mind a couple of spaces between either side of the `+`. After that feel free to merge the PR yourself.